digdag-standards/src/main/java/io/digdag/standards/operator/jdbc/LockConflictException.java

```diff
@@ -0,0 +1,28 @@
+package io.digdag.standards.operator.jdbc;
+
+import java.sql.SQLException;
+
+public class LockConflictException
+        extends RuntimeException
```

Comment:
Oops, good catch!

Actually, I was planning to make it extend Exception and added definition `throws LockConfilictException`... https://github.com/treasure-data/digdag/pull/198/files/d798bc0b3283477b5f3179c0c07b804eac59822f#diff-6796cda912b3d95467f3864be5e93189R120